src/containerapp/azext_containerapp/tests/latest/test_containerapp_create.py

```diff
@@ -0,0 +1,46 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+import unittest
+
+from azure.cli.testsdk import (ScenarioTest, ResourceGroupPreparer, live_only)
+
+from azext_containerapp.tests.latest.utils import create_and_verify_containerapp_create
+
+TEST_DIR = os.path.abspath(os.path.join(os.path.abspath(__file__), '..'))
+
+class ContainerAppCreateTest(ScenarioTest):
+    @live_only()
```

Comment:
avoid live_only if the tests are replayable

Reply:
[This test](https://github.com/snehapar9/azure-cli-extensions/blob/snehapar/Add-source-and-repo-to-create/src/containerapp/azext_containerapp/tests/latest/test_containerapp_create.py#L26) needs to be a live test because build packs requires docker to be running.